Gemfile

```diff
@@ -17,10 +17,10 @@ end
 # We don't put beaker in as a test dependency because we
 # don't want to create a transitive dependency
 group :acceptance_testing do
-  gem "beaker", *location_for(ENV['BEAKER_VERSION'] || '~> 2.0')
+  gem "beaker", *location_for(ENV['BEAKER_VERSION'] || '~> 4')
 end
 
 
-if File.exists? "#{__FILE__}.local"
+if File.exist? "#{__FILE__}.local"
   eval(File.read("#{__FILE__}.local"), binding)
 end
```

Comment:
I have a tendency to remove these blocks to reduce chaos and/or arbitrary-code-injection. Keep it around if you know of a good case for it but I would mildly suggest deleting it otherwise.